ViewGenerator/ViewGenerator.csproj

```diff
@@ -16,6 +16,7 @@
     </NuGetPackageImportStamp>
     <TypeScriptToolsVersion>2.6</TypeScriptToolsVersion>
     <TargetFrameworkProfile />
+    <CleanFilesExcludePath Condition=" '$(CleanFilesExcludePath)' == '' ">Legacy\</CleanFilesExcludePath>
```

Comment:
This shouldn't be here, please remove it (only affects ViewGenerator build)

Reply:
You have to add this on your plugin's csproj later

Reply:
Btw I would assume the default CleanFilesExcludePath to empty (undefined). I think we won't need it in most projects using ViewGenerator

Reply:
Thanks. I added it to the plugin project but then remembered to add it here conditionally thinking it would act as a default if project didn't define it. To handle the cases where it is not defined to not break the JSFilesToDelete Exclude parameter. I'll do the change and also fix the condition on JSFilesToDelete